### PR TITLE
Enable VB Semantic tests on CoreClr Linux

### PR DESCRIPTION
**Pull request by jaredpar**
Part of #29399

### CODE REVIEW

**Comment by jaredpar** [comment]
CC @dotnet/roslyn-compiler for review

File: src/Test/Utilities/Portable/TestBase.cs [hunk 1]
```diff
@@ -362,9 +362,24 @@ private static MetadataReference GetOrCreateMetadataReference(ref MetadataRefere
...
+            if (ExecutionConditionUtil.IsWindows)
+            {
+                if (!inputString.Contains("\r\n") && inputString.Contains("\n"))
+                {
+                    return inputString.Replace("\n", "\r\n");
+                }
+            }
+            else
+            {
+                if (inputString.Contains("\r\n"))
```
Comment (cston) [inline src/Test/Utilities/Portable/TestBase.cs]: `if` check is not necessary.

Comment (jaredpar) [reply src/Test/Utilities/Portable/TestBase.cs]: Will fix that up in the upcoming PR to enable Emit tests.

**Comment by jaredpar** [comment]
retest ubuntu_16_debug_prtest please

File: src/Test/Utilities/Portable/TestBase.cs [hunk 2]
```diff
@@ -353,21 +353,21 @@ private static MetadataReference GetOrCreateMetadataReference(ref MetadataRefere
...
                 startLocation,
                 syntaxNodePredicate,
                 argumentOrderDoesNotMatter);
         }
 
-        protected static string NormalizeDiagnosticString(string inputString)
+        public static string NormalizeNewLines(XCData data)
         {
-            if (!inputString.Contains("\r\n") && inputString.Contains("\n"))
+            if (ExecutionConditionUtil.IsWindows)
```
Comment (333fred) [inline src/Test/Utilities/Portable/TestBase.cs]: Isn't this going to break any text that comes in with `\r\n`? We'll end up with `\r\r\n`.

Comment (jaredpar) [reply src/Test/Utilities/Portable/TestBase.cs]: The `XCData` type is the input here and it can only have `\n` in the `Value` property. Hence this is safe

**Pull request merged**